cla_public/apps/checker/fields.py

```diff
@@ -161,10 +161,22 @@ def __init__(self, label=None, validators=None, min_val=0, max_val=9999999999, *
         self.min_val = min_val
         self.max_val = max_val
 
+    def convert_amount_to_utf8(self, amount):
+        # Covert pounds side of the input into UTF-8 format
```

Comment:
There's nothing in this method which makes it as specific as this comment suggests. It could be used for any text value, so I think the comment is somewhere between redundant and misleading. The comment would make sense (but also be redundant!) in the place this function is used.
If you wanted to document what this function does, I think it should better reflect what the function actually does, rather than its one current use case that it's not particularly specialised for.